Source/core/DoorBell.cpp

```diff
@@ -34,7 +34,11 @@ namespace Core {
     DoorBell::Connector::Connector(DoorBell& parent, const Core::NodeId& node)
         : _parent(parent)
         , _doorbell(node)
+        #ifdef __LINUX__
```

Comment:
The SOCK_CLOEXEC is also available on windows so no need to use the preprocessor to diffrentiate.